packages/paste-website/src/component-examples/FormPillGroup.tsx

```diff
@@ -124,3 +185,192 @@ render(
   <I18nFormPillGroup />
 )
 `.trim();
+
+export const selectedStateExample = `
+const SelectedStateExample = () => {
+  const pillState = useFormPillState();
+
+  return (
+    <form>
+      <FormPillGroup {...pillState} aria-label="Products:">
+        <FormPill {...pillState} selected>
+         Voice
+        </FormPill>
+        <FormPill {...pillState} selected>
+          <ProductVideoIcon decorative />
+          Video
+        </FormPill>
+        <FormPill {...pillState} selected>
+          <ProductVerifyIcon decorative />
+          Verify
+        </FormPill>
+      </FormPillGroup>
+    </form>
+  );
+};
+
+render(
+  <SelectedStateExample />
+)
+`.trim();
+
+export const errorStateExample = `
+const ErrorStateExample = () => {
+  const pillState = useFormPillState();
+
+  return (
+    <form>
+      <FormPillGroup {...pillState} aria-label="Products:">
+        <FormPill {...pillState} variant="error">
+         Voice
+        </FormPill>
+        <FormPill {...pillState} variant="error" selected>
+          <ProductVideoIcon decorative />
+          Video
+        </FormPill>
+        <FormPill {...pillState} variant="error">
+          <ProductVerifyIcon decorative />
+          Verify
+        </FormPill>
+        <FormPill {...pillState} variant="error" selected>
+         Usage
+        </FormPill>
+      </FormPillGroup>
+    </form>
+  );
+};
+
+render(
+  <ErrorStateExample />
+)
+`.trim();
+
+export const disabledStateExample = `
+const DisabledStateExample = () => {
+  const pillState = useFormPillState();
+
+  return (
+    <form>
+      <FormPillGroup {...pillState} aria-label="Products:">
```

Comment:
Todo: at least one (or all?) of these `FormPill`s need to include the `disabled` prop in this example

Reply:
Ack! Thanks for catching that. Fixed.